apps/framework-cli/src/cli.rs

```diff
@@ -152,6 +161,37 @@ async fn top_command_handler(settings: Settings, commands: &Commands) {
                 controller.add_routine(Box::new(CleanProject::new(project_arc, run_mode)));
                 controller.run_routines(run_mode);
             }
+            Commands::Docker { sub_method } => match sub_method.as_str() {
```

Comment:
I'm pretty sure there's a clean enum based approach here as opposed to raw string matching. Not a blocker but something you may want to look into as you keep moving this forward